enhancements/installer/image-based-installer.md

```diff
@@ -0,0 +1,331 @@
+  of the OpenShift installer, or a standalone binary?
+
+  Having the functionality provided by the command-line tool in the OpenShift
+  installer would be a natural addition to the latter, as the former refers to
+  the provisioning of single-node OpenShift clusters and consumes the OpenShift
+  installer `install-config.yaml`. It generates the required installation
+  artifacts in the same way as the [Agent-based Installer](/enhancements/agent-installer/agent-based-installer.md)
+  `openshift-install agent create config-image` command.
+
+## Test Plan
+
+The Image-based Installer will be covered by end-to-end testing using virtual
+machines (in a baremetal configuration), automated by some variation on the
+metal platform [dev-scripts](https://github.com/openshift-metal3/dev-scripts/#readme).
+This is similar to the testing of the Agent-based Installer, the baremetal IPI
+and assisted installation flows.
+
+## Graduation Criteria
+
+**Note:** *Section not required until targeted at a release.*
+
+TBD
+
+### Dev Preview -> Tech Preview
+
+- Ability to utilize the enhancement end to end
+- End user documentation, relative API stability
+- Sufficient test coverage
+- Gather feedback from users rather than just developers
+- Enumerate service level indicators (SLIs), expose SLIs as metrics
+- Write symptoms-based alerts for the component(s)
+
+### Tech Preview -> GA
+
+- More testing (upgrade, downgrade, scale)
+- Sufficient time for feedback
+- Available by default
+- Backhaul SLI telemetry
+- Document SLOs for the component
+- Conduct load testing
+- User facing documentation created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+**For non-optional features moving to GA, the graduation criteria must include
+end to end tests.**
+
+### Removing a deprecated feature
+
+N/A
+
+## Upgrade / Downgrade Strategy
+
+N/A
+
+## Version Skew Strategy
+
+### Configuration ISO version and Seed OCI Image version
+
+The IBI configuration ISO content version should be compatible with the seed
+OCI image version contained in the IBI installation ISO. The Lifecycle Agent
+Operator, which is contained in the installation ISO, is responsible for
+ensuring the compatibility of those two at the time of the image-based
+installation. The configuration version does not depend on the OpenShift
+versions used to generate either one of the IBI artifacts (i.e. configuration
+ISO and installation ISO). In the case of incompatibility between the
+configuration version and the seed OCI image version, the image-based
+installation will fail with the respective error message.
+
+### Installation ISO RHCOS ISO version and Seed OCI Image version
+
+The RHCOS base ISO, which is contained in the IBI installation ISO and derived
+from the OpenShift release payload, has no strict requirements to be tied to the
+seed OCI image OpenShift version. The features and configuration of the
```

Comment:
Is this because the ostree image that we write to the disk is derived from the seed image, and _not_ the rootfs of the live ISO that is doing the installation?

Reply:
Yes.